tensorboard/plugins/mesh/mesh_demo.py

```diff
@@ -0,0 +1,115 @@
+"""Simple demo which displays constant 3D mesh."""
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Demo application for Mesh Plugin."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+import numpy as np
+import tensorflow as tf
+import trimesh
+
+from tensorboard.plugins.mesh import summary as mesh_summary
+
+tf.flags.DEFINE_string('logdir', '/tmp/mesh_demo',
```

Comment:
Can we use `absl.flags` instead of `tf.flags`, as elsewhere in
TensorBoard? You can `git grep 'absl.*flags'` for examples.